src/Upgrader/Report/Service/ReportService.php

```diff
@@ -71,8 +83,15 @@ public function report(): ?array
         $messageSeparator = str_pad('', 100, '-');
         foreach ($output[static::KEY_VIOLATIONS] as $violations) {
             $key = $violations[static::KEY_PRODUCED_BY];
-            $keySeparatorLength = str_pad('', strlen($key), '-');
-            $messages[] = $key . ' ' . $violations[static::KEY_MESSAGE] . PHP_EOL . $keySeparatorLength . ' ' . $messageSeparator;
+            $message = $key . ' ' . $violations[static::KEY_MESSAGE] . PHP_EOL;
```

Comment:
From my perspective, better to add `getMessage` method 

`    public function report(bool $isVerbose = false): ?array
    {
        $path = getcwd() . sprintf(static::FILE_PATH_PATTERN, AnalyzeTask::ID_ANALYZE_TASK);

        if (!file_exists($path)) {
            return null;
        }

        $output = Yaml::parseFile($path);

        $messages = [];
        $messageSeparator = str_pad('', 100, '-');
        foreach ($output[static::KEY_VIOLATIONS] as $violation) {
            $messages[] = sprintf('%s %s', $this->getMessage($violation, $isVerbose), $messageSeparator);
        }

        return $messages;
    }

    /**
     * @param array $violations
     * @param bool $isVerbose
     * @return string|null
     */
    protected function getMessage(array $violations, bool $isVerbose = false): ?string
    {
        $key = $violations[static::KEY_PRODUCED_BY];
        $message = $key . ' ' . $violations[static::KEY_MESSAGE] . PHP_EOL;

        if ($isVerbose) {
            $docUrl = $violations[static::KEY_ADDITIONAL_ATTRIBUTES][static::KEY_ATTRIBUTE_DOCUMENTATION] ?? '';
            $message .= str_pad('', strlen($key) + 1, ' ') . '💡More information: ' . $docUrl . PHP_EOL;
        }

        $message .= str_pad('', strlen($key), '-');
        
        return $message;
    }`